numpy/core/tests/test_dtype.py

```diff
@@ -13,6 +13,8 @@
 from numpy.compat import pickle
 from itertools import permutations
 
+IS_PYSTON = hasattr(sys, "pyston_version_info")
+
```

Comment:
Can you import this from numpy.testing like `HAS_REFCOUNT` (here and in the other files as well)?